.env.example

```diff
@@ -9,3 +9,21 @@ REACT_APP_SUPPORT_URL=
 REACT_APP_SENTRY_DSN_URL=
 REACT_APP_SENTRY_ENV=development
 SENTRY_AUTH_TOKEN=
+
+## Chainbridge configuration
+### Ethereum 
+REACT_APP_ETHEREUM_CHAIN_ID="5"
+REACT_APP_ETHEREUM_NETWORK_ID="5"

```

Comment:
What is NETWORK ID?

Reply:
The network id is a unique Id for all networks with Ethereum. For Goerli it is 5